docs/manage/data-archiving/index.md

```diff
@@ -0,0 +1,39 @@
+---
+slug: /manage/data-archiving
+title: Data Archiving
+description: Data Archiving allows you to archive selected data to an external server or Amazon S3.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/icons/operations/send-data.png')} alt="icon" width="50"/>
+
+Archive allows you to forward log data from Installed Collectors to AWS S3 buckets to collect at a later time. If you have logs that you don't need to search immediately you can archive them for later use. You can ingest from your Archive on-demand with five-minute granularity.  
```

Comment:
```suggestion
Archive allows you to forward log data from Installed Collectors to AWS S3 buckets to collect at a later time. If you have logs that you don't need to search immediately, you can archive them for later use and ingest from your Archive on-demand with five-minute granularity.  
```

Reply:
Thanks @kimsauce I have slightly tweaked this sentence and split the last sentence.